lib/src/pagination/pagination.component.html

```diff
@@ -0,0 +1,101 @@
+<div class="pagination-wrapper" [ngClass]="className" [attr.id]="id">
+    <nav class="custom-pagination">
```

Comment:
since there is no custom css for the pagination wrapper and custom pagination I see them as unnecessary wrappers which should be removed

Reply:
I think you should leave this class for now until I refactor the React version to use Bootstrap markup. Once I'm done, I'll be making the necessary changes. No need to waste time on it now